apis/dedicatedelb/v1beta1/zz_member_types.go

```diff
@@ -61,6 +61,7 @@ type MemberParameters struct {
 	// must be TCP, HTTP, or HTTPS.
 	// The IPv4 or IPv6 subnet ID of the subnet in which to access the member
 	// +crossplane:generate:reference:type=github.com/FrangipaneTeam/provider-flexibleengine/apis/vpc/v1beta1.VPCSubnet
+	// +crossplane:generate:reference:extractor=github.com/FrangipaneTeam/provider-flexibleengine/pkg/tools.ExtractorParamPathfunc(true, "ipv4_subnet_id")
```

Comment:
For `flexibleengine_lb_member_v3`, `subnet_id` can be `ipv4_subnet_id` OR `ipv6_subnet_id`